### DISCUSSION TITLE
Override the 45 minute build limit?

### DISCUSSION THREAD

**Discussion by amanda-gordon** [vercel/community#1313] [Help]
### Summary

Hi - Is it possible to override the Vercel 45 minute build timeout? I would like to set the timeout to 30 minutes instead of 45. I have checked the following documentation, but can't find any guidance about this:

Vercel - Configuring a Build: [https://vercel.com/docs/concepts/deployments/configure-a-build](https://vercel.com/docs/concepts/deployments/configure-a-build)

Vercel - Limits (see Build Time per Deployment): [https://vercel.com/docs/concepts/limits/overview](https://vercel.com/docs/concepts/limits/overview)

**Comment by leerob** [comment]
Hey there, this isn't possible to be configured currently. Could you share more about why you're looking to limit?

**Comment by amanda-gordon** [comment]
Hi @leerob thanks for getting back to me. Got it. We have certain users able to use build hooks in Drupal to trigger builds, and then they monitor those builds through Slack integration. Once a particular build hits 20+ minutes, we know that the build will fail. However, since those particular users cannot kill builds, they have to wait an additional 25 minutes for the build to timeout before triggering a new one.
